src/wifi/model/basic-dmg-wifi-scheduler.cc

```diff
@@ -267,7 +267,7 @@ BasicDmgWifiScheduler::AddBroadcastCbapAllocations (void)
     {
       start = iter->GetAllocationStart () + iter->GetAllocationBlockDuration () + m_guardTime;
       if ((m_remainingDtiTime >= m_interAllocationDistance)
-          && (m_interAllocationDistance > 0)) // here the decision to place a broadcast CBAP among allocated requests
+          && (m_interAllocationDistance > m_guardTime)) // here the decision to place a broadcast CBAP among allocated requests
```

Comment:
m_interAllocationDistance is an attribute. Why should it be checked every time? Instead, getter/setter could be added to check m_interAllocationDistance > m_guardTime
Furthermore, isn't m_guardTime a constant given by the standard? If so, maybe it could be modified into static const in DmgWifiScheduler

Reply:
if it's equal to the guardTime it means we are not placing a CBAPs among SPs